example/src/main/java/io/realm/examples/adapters/model/Counter.java

```diff
@@ -15,19 +15,30 @@
  */
 package io.realm.examples.adapters.model;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
 import io.realm.RealmObject;
 
-public class TimeStamp extends RealmObject {
+public class Counter extends RealmObject {
+    public static final String FIELD_COUNT = "count";
+
+    private static AtomicInteger integerCounter = new AtomicInteger(0);
 
-    public static final String TIMESTAMP = "timeStamp";
+    private int count;
 
-    private String timeStamp;
+    public int getCount() {
+        return count;
+    }
+
+    public void setCount(int count) {
+        this.count = count;
+    }
 
-    public String getTimeStamp() {
-        return timeStamp;
+    public String getCountString() {
+        return Integer.toString(count);
     }
 
-    public void setTimeStamp(String timeStamp) {
-        this.timeStamp = timeStamp;
+    public void setAndIncrease() {
```

Comment:
I think `increment` is probably a more reflective name, as we're not setting any value (method doesn't take args)